### DISCUSSION TITLE
Garage Spawn Dir

### DISCUSSION THREAD

**Discussion by TheLukio** [Vdauphin/HeartsAndMinds#1458] [Question and answer]
Is it possible to make the spawn direction the same as the dir of the "btc_log_point"  marker?

Currently vehicles will only be created in dir 0 (default) which depending on location is a bit of headache at times (small / congested bases or airstrips for example). Being able to adjust the direction the vehicle spawns just by adjusting the dir of the marker would be a simple solution to this issue.

For now we've enabled towing for planes as a workaround.

**Comment by Vdauphin** [comment]
Hello,
I made a patch https://github.com/Vdauphin/HeartsAndMinds/pull/1459
Cheers
